docs/security/content_trust.md

```diff
@@ -0,0 +1,156 @@
+<!--[metadata]>
++++
+alias = [TBD]
+title = "Content trust in Docker"
+description = "Enabling content trust in Docker"
+keywords = ["trust, security, docker,  documentation"]
+[menu.main]
+parent= "smn_administrate"
++++
+<![end-metadata]-->
+
+## Enabling content trust in Docker
+
+Docker 1.8 enables content trust. Content trust means that all operations with a remote registry enforce  client-side signing and verification of content tags. Currently content trust is opt-in. We are planning to change this behavior to opt-out in a future release. For now, in order to have trusted operations with Docker you will need to either `export DOCKER_CONTENT_TRUST=1` , or use the `--disable-content-trust=false` flag.
+
+When enabled, content trust will ensure that all operations using a remote registry enforce the signing and verification of tags. In particular `push`, `pull`, `build`, `create` and `run` will only operate on tags that either have content signatures or explicit content hashes.
+
+#### Pulling trusted content
+Let's try to pull a remote repository on the Docker hub that does not have content-trust data:
+
+```
+➜ export DOCKER_CONTENT_TRUST=1
+➜ docker pull diogomonica/alpine
```

Comment:
could you change the `➜` to `$` in your examples, to be consistent with the rest of the documentation?